webui/react/src/pages/TrialDetails/TrialDetailsHyperparameters.tsx

```diff
@@ -0,0 +1,28 @@
+import { List } from 'antd';
+import React from 'react';
+
+import Section from 'components/Section';
+import { ExperimentBase, TrialDetails } from 'types';
+import { isObject } from 'utils/data';
+
+export interface Props {
+  experiment: ExperimentBase;
+  trial: TrialDetails;
+}
+
+const TrialDetailsHyperparameters: React.FC<Props> = ({ trial }: Props) => {
+  return (
+    <Section bodyBorder bodyNoPadding>
+      <List
```

Comment:
@hkang1 we could consider displaying those in a Table instead of a List. Do you think that's better?

In case:
- I guess the table should not be sortable?
- What headers could we use for the 2 colums? "Hyperparameter" and "Value"?

Reply:
@apizzini I think that's a great idea. The 2 columns you proposed sounds good, and then make just `Hyperparameter` column sortable (as alphabetical)